ExampleApp/package.json

```diff
@@ -28,7 +28,7 @@
       "android"
     ],
     "plugins": {
-      "optimove-cordova-plugin": {}
+      "cordova-sdk": {}
```

Comment:
With it being scoped, is there any risk of collisions I wonder.. Can/should we prefix in the 'id', but not 'name' fields of meta-data? (if this is even possible..)

Reply:
suppose no reason not to, updated